src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,29 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+
+    private String driveType;
+    private int loadCapacity;
```

Comment:
The access to the fields `driveType` and `loadCapacity` should be restricted using access modifiers. Consider making these fields private to follow the encapsulation principle.